### PR TITLE
Fix the durability when sending messages to a non-durable AMQP address, enforced with reactive-messaging 2.2

### PR DESCRIPTION
**Pull request by rsvoboda**
Fix the durability when sending messages to a non-durable AMQP address, enforced with reactive-messaging 2.2

Fixes https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/66

Introduced in https://github.com/quarkusio/quarkus/commit/0dd61af3e6a7e6d92f121d53ebb85b429416ffce

### CODE REVIEW

File: messaging/amqp-reactive/src/main/resources/application.properties [hunk 1]
```diff
@@ -11,7 +11,6 @@ amqp-port=5672
 # Configure the AMQP connector to write to the `prices` address
 mp.messaging.outgoing.generated-price.connector=smallrye-amqp
 mp.messaging.outgoing.generated-price.address=prices
-mp.messaging.outgoing.generated-price.durable=true
```
Comment (Ladicek) [inline messaging/amqp-reactive/src/main/resources/application.properties:14]: What I find really weird, at least from my perspective (I don't know almost anything about messaging) is that you delete the `durable` setting for `outgoing`, but not for `incoming`, even if they address the same queue.

Comment (Ladicek) [reply messaging/amqp-reactive/src/main/resources/application.properties:14]: Hmm, or they don't use the same queue? The comments suggest they do, but `incoming` doesn't have the `address` setting at all. Oh well.

Comment (rsvoboda) [reply messaging/amqp-reactive/src/main/resources/application.properties:14]: I'm asking for some input on https://github.com/quarkusio/quarkus/commit/0dd61af3e6a7e6d92f121d53ebb85b429416ffce feel free to comment there too

Same change is in quarkus-quickstarts ....

Comment (Ladicek) [reply messaging/amqp-reactive/src/main/resources/application.properties:14]: Thanks.

**Pull request merged**